packages/data/lib/database/database_provider.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:domain/model/movie_db_model.dart';
+import 'package:domain/model/people_and_images_model.dart';
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+
+class DataBaseProvider {
+  static final DataBaseProvider instanse = DataBaseProvider._init();
+
+  static const String dbName = 'movie_list.db';
+  static const String movieTableName = 'Movies';
+  static const String castTableName = 'Cast';
+  static const int dbVersion = 1;
+  static Database? _database;
+
+  DataBaseProvider._init();
+
+  Future<Database> get database async {
+    if (_database != null) return _database!;
+
+    _database = await _initDB(dbName);
+    return _database!;
```

Comment:
```suggestion
    if (_database == null) _database = await _initDB(dbName);
    return _database!;
```

the same, but more compact

Reply:
okay